### DISCUSSION TITLE
How do I know the parameters that can be selected

### DISCUSSION THREAD

**Discussion by RewJiamo** [open-mmlab/mmdetection#9099] [Q&A] [answered]
Such as：
evaluation = dict( 
    interval=1,
    metric=['bbox', 'segm']
)

How can I get all choose for "metric"? such as I want to use MR^-2 to evaluation,but I don't if metric has it.

And some thing else, how to set to not use the "pretrained" in backbone?

**Comment by ZwwWayne** [comment, marked as answer]
You can simply set init_cfg=None or pretrained=None in the config.
For evaluation, it corresponds to the evalhook. It essentially defines the parameters in https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/evaluation/eval_hooks.py#L26
